### ISSUE TITLE
Bad item "home" found in filesystem overrides; Bad item "host" found in filesystem overrides

### ISSUE THREAD

**Issue by sanecito** [flathub/com.valvesoftware.Steam#1091]
**Game information**

N/A

**Distribution name and version where applicable**

Fedora Kinoite 38.20230601.0

**Flatpak info**

Flatpak 1.15.4

Flatpak GL-Drivers:
default
host

**Problem description**

Issue started in the last week or so. When launching Steam it warns about overrides I haven't touched and won't launch Steam. I deleted the flatpak and all Steam data including user data via Discover's wipe all related data feature to make sure I wiped any overrides I might have absently mindly created in the far past for good measure, but still get the errors after reinstall and launch attempts.

```
flatpak run com.valvesoftware.Steam
INFO:root:https://github.com/flathub/com.valvesoftware.Steam/wiki
WARNING:root:Bad item "home" found in filesystem overrides
WARNING:root:Bad item "host" found in filesystem overrides
Please see https://github.com/flathub/com.valvesoftware.Steam/wiki#i-want-to-add-external-disk-for-steam-libraries
```

**Does this issue reproduce with native Steam**

Have not tested

**Comment by MiranDaniel** [comment]
Also having this issue on openSUSE Tumbleweed with Flatpak 1.14.4. Native Steam works fine.

**Comment by vostronix** [comment]
Same here

**Comment by sanecito** [comment]
I don't recall ever touching Flatpak settings, apart from another app, RetroDeck. However, when I looked at Flatpak privileges via Flatseal there were some non-default values for Steam. Resetting it fixed this issue for me.

![image](https://github.com/flathub/com.valvesoftware.Steam/assets/3056597/6a0c728d-6d50-4474-a17e-84369f57d2bd)

**Comment by nanonyme** [comment]
Anyone having this should double-check if they have global overrides that also affect this app. Sandbox holes that expose host user home are not supported.

**Comment by sanecito** [comment]
So in doing a new system build on Fedora Silverblue and Kinoite again, I discovered I may have actually been the one to have changed the default flatpak permissions. I couldn't get controller input to be recognized within games (worked fine in Big Picture mode) in wired or wireless modes. The solution to my controller issue was to enable access to `D-Bus system bus` / `socket=system-bus` in Flatseal. It's possible I didn't do this, but I tried a relatively exhaustive list of things like making sure of things like Steam settings around controllers, and various suggestions found in https://github.com/flathub/com.valvesoftware.Steam/issues/734

As I'm mildly confident this is what happened which is technically deviating from the official flatpak permissions that causes this issue, I'm closing my issue. For those also deviating to get Steam controller support, I suggest pushing for better controller support in one of the several open controller issues
